core/src/main/resources/hudson/model/View/sidepanel_pl.properties

```diff
@@ -22,7 +22,7 @@
 
 Build\ History=Historia zada\u0144
 Check\ File\ Fingerprint=Sprawd\u017A Odcisk Palca Pliku
-Delete\ View=Usu\u0144 Widok
+Delete\ View=Usu\u0144 widok
```

Comment:
Name Of The Company,  Name Of The Institution but this is just command and it looked strange.  There are also other similar mistakes but they couldn't be fixed by translation.  They are caused by wrong conception so I will take a look and check if this is big effort to implement such improvement.